Misc/NEWS.d/next/Documentation/2020-11-15-13-46-31.bpo-42153.KjBhx3.rst

```diff
@@ -0,0 +1,3 @@
+Source Code of Documents describing the protocol, sources and binaries for servers
+implementing it, by the University of Washington's IMAP Information Center
+can all be found at (https://github.com/uw-imap/imap) (**Not Maintained**).
```

Comment:
Usually small doc fixes like this don't need a news entry; if you really want a news entry, it should just read "Fix the URL for the IMAP protocol documents."

Reply:
Isn't NEWS ENTRY required. The check fails when news entry is removed.